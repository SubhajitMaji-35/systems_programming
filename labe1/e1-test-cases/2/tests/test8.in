-2 3 7 -5
7 3 0 -7
5 0 -8 -4
2 3 6 9
