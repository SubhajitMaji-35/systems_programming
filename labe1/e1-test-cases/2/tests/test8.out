-2 7 5 2 
3 3 0 3 
7 0 -8 6 
-5 -7 -4 9 
